config/webpack.config.js

```diff
@@ -34,23 +33,22 @@ function generateConfig (name) {
   const config = {
     mode,
     entry: {
-      circuitBreaker: './index.js'
+      CircuitBreaker: './index.js'
     },
     output: {
       path: path.resolve(__dirname, '..', 'dist'),
       filename: `${name}.js`,
       sourceMapFilename: `${name}.map`,
-      library: 'circuitBreaker',
+      library: 'CircuitBreaker',
       libraryTarget: 'umd'
     },
     node: {
       process: true,
       console: true
     },
     plugins: [
-      new webpack.IgnorePlugin(/prom-client/),
       new webpack.ProvidePlugin({
-        'circuitBreaker': 'opossum'
+        'CircuitBreaker': 'opossum'
```

Comment:
```suggestion
        'CircuitBreaker': 'opossum',
        'circuitBreaker': 'opossum'
```

Not 100% sure, but I think this will keep backwards compatibility.

Reply:
If that works, then that would be a good way to do it

Reply:
@lance anymore thoughts on this?

I would like to keep things consistent, especially for the examples,  but it is not totally needed

Reply:
Sorry - I looked at this last week and just really couldn't decide. I think it will need to be at least a minor version bump (and probably should be major). I am inclined to keep it as-is to avoid having such a small change cause a big version bump.

Reply:
that sounds reasonable.   I will keep this open though, so when we do have a bigger change that will require a major change, maybe that is when we add this

Reply:
@lance in a recent commit we dropped node 8 support, 9145c9263118dc0ccbe5d2621373e5d0f801ed60 , so i was going to do a major release for it.  I'm thinking about getting this added, if we are going to do a major release.  It might make sense to do now.  any thoughts?

I'm still not totally sure about it though,  since it is just a coding convention

Reply:
Fine with me.